apps/sms/views/shared/style/panel.css

```diff
@@ -0,0 +1,65 @@
+.panel {
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+
+  display: none;
+
+  overflow: auto;
+
+  background: white;
+
+  animation: .2s both linear;
+  transform-origin: top;
+}
+
+.panel-active {
+  display: unset;
+}
+
+.panel-prepare-activation {
+  display: unset;
+  visibility: hidden;
+}
+
+.panel-will-activate {
+  animation-name: new-slide-left;
+}
+
+.panel-will-activate.panel-animation-reverse {
+  animation-name: new-slide-right;
+}
+
+.panel-will-deactivate {
+  animation-name: old-slide-left;
+}
+
+.panel-will-deactivate.panel-animation-reverse {
+  animation-name: old-slide-right;
+}
+
+@keyframes new-slide-left {
+  from {
+    transform: translateX(100%);
+  }
+}
+
+@keyframes old-slide-left {
+  to {
+    transform: translateX(-100%);
+  }
+}
+
+@keyframes new-slide-right {
+  from {
+    transform: translateX(-100%);
+  }
+}
+
+@keyframes old-slide-right {
+  to {
+    transform: translateX(100%);
+  }
```

Comment:
I removed the fancy transitions but they're still at https://github.com/mozilla-b2g/gaia/commit/f831d6925298894c20612adbec5b796ec3822e69 :)